src/ScottPlot5/ScottPlot5/Rendering/RenderManager.cs

```diff
@@ -146,8 +147,25 @@ public void Render(SKCanvas canvas, PixelRect rect)
             }
         }
 
+        NeedsAnotherRender = CheckAxesChangedByEvents();
```

Comment:
We probably want to opt into this behaviour. Perhaps only process the path if `EnableEvents `is true?